src/components/private-name-tag/dtos/get-private-name-tag-result.dto.ts

```diff
@@ -9,11 +9,20 @@ class PrivateNameTagResponseAttributes {
   })
   address: string;
 
+  @ApiProperty({ example: false })
+  is_favorite: boolean;
+
```

Comment:
chỗ này chuyển thành dạng camelcase được k a
```suggestion
  @ApiProperty({ example: false, name: is_favorite })
  isFavorite: boolean;

```
tương tự những thằng ở dưới